static/data/filters/color-filter.json

```diff
@@ -0,0 +1,19 @@
+{
+	"filters": [
+		{
+			"label": "R",
+			"color": "red",
+			"img": "../../../../static/images/red-filter.png"
+		},
+		{
+			"label": "B",
+			"color": "blue",
+			"img": "../../../../static/images/blue-filter.png"
+		},
+		{
+			"label": "G",
+			"color": "green",
+			"img": "../../../../static/images/green-filter.png"
+		}
+	]
+}
```

Comment:
Your image paths do not need to include the `../../../../static` bit. You'll just need `[image-file-name]` and finish the image path in your component.